.github/workflows/check-pull-request.yml

```diff
@@ -0,0 +1,26 @@
+name: Check Branch
```

Comment:
What do you think about renaming this and the file name to `name: Test` and `test.yml`

Reply:
Sounds good to me @gabrieltaylor